packages/gestalt/src/Popover/Contents.js

```diff
@@ -0,0 +1,133 @@
+// @flow strict
+import { type Node, useEffect, useRef } from 'react';
+import { FloatingFocusManager } from '@floating-ui/react';
+import classnames from 'classnames';
+import usePopover, { DIRECTIONS_MAP, SIDES_MAP } from './usePopover.js';
+import borders from '../Borders.css';
+import Caret from '../Caret.js';
+import colors from '../Colors.css';
+import styles from '../Contents.css';
+import { type MainDirections } from '../utils/positioningTypes.js';
+import { CARET_HEIGHT, CARET_WIDTH } from '../utils/positioningUtils.js';
+
+export type Role = 'dialog' | 'listbox' | 'menu' | 'tooltip';
+
+type Props = {
+  accessibilityLabel?: string,
+  anchor: HTMLElement,
+  bgColor: 'blue' | 'darkGray' | 'orange' | 'red' | 'white',
+  border?: boolean,
+  caret?: boolean,
+  children?: Node,
+  id: ?string,
+  idealDirection?: MainDirections,
+  onKeyDown: (event: SyntheticKeyboardEvent<HTMLElement>) => void,
+  role: ?Role,
+  rounding?: 2 | 4,
+  shouldFocus?: boolean,
+  width: ?number,
+  scrollBoundary?: HTMLElement,
+};
+
+export default function Contents({
```

Comment:
Can we change name here or in File to  NewContents.

I know locations / paths are different but I want to make sure we dont exhange them by misatke. we cant rerename in the future

Reply:
What about Experimental prefix? Or we can rename old ones with prefix Legacy.

Reply:
I'll leave it up to you. Legacy maybe makes more sense?

Reply:
Renamed old one with `Legacy` prefix